app/waf/src/controllers/application.controller.ts

```diff
@@ -172,6 +177,160 @@ export class ApplicationController extends BaseController {
     });
   }
 
+  @post(prefix + '/applications/{applicationId}/append', {
```

Comment:
Why not reusing deploy/cleanup actions? I feel we may only provide two action APIs for end users, one is to deploy configuration, and the other is to clean the deployed configuration. End user may not need to know the difference between 'append' and 'replace'.

Reply:
That's ok and will greatly simplify the codes...

Reply:
I will make the changes accordingly.